dashboard/src/features/tutorial/components/Tutorial.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import steps from './steps.json'
+import Description from './Description/Description'
+import Success from './Success/Success'
+import Form from './Form/Form'
+import Complete from './Complete/Complete'
+
+const components = {
+  Description,
+  Success,
+  Form,
+  Complete
+}
+
+class Tutorial extends React.Component {
+  render() {
+    const tutorialStep = this.props.tutorialStep
+    const userInput = this.props.tutorialInputs
+    const tutorialOpen = this.props.tutorialOpen
+    const tutorialTypes = this.props.types
+    const TutorialComponent = components[steps[tutorialStep]['component']]
+
+    return (
+      <div>
+      {tutorialOpen && (tutorialTypes.includes(steps[tutorialStep]['component'])) &&
+          <TutorialComponent
+            userInput={userInput}
+            step={tutorialStep}
+            button={steps[tutorialStep]['button']}
+            title={steps[tutorialStep]['title']}
+            content={steps[tutorialStep]['content']}
+            dismiss={steps[tutorialStep]['dismiss']}
+            route={steps[tutorialStep]['route']}
```

Comment:
If it doesn't cause problems, you can splat `steps[tutorialStep]` as component props:

```jsx
<TutorialComponent
  userInput={userInput}
  step={tutorialStep}
  {...steps[tutorialStep]}
  handleNext={this.props.showNextStep}
  handleDismiss={this.props.dismissTutorial}
/>
```

Not committed to that one way or another; it will add at least one unintended property, `component`, to the component too.